content/foundations/color.mdx

```diff
@@ -17,21 +17,30 @@ import {Box, Button, Heading, Label, LabelGroup, Link} from '@primer/react'
   src="https://user-images.githubusercontent.com/6951037/146927448-cb518377-114a-4ab8-a37e-1eeffc8732c7.png"
 />
 
-GitHub's UI offers a variety of different color modes. When designing product interfaces, you should design in light mode by default. Every pattern in Primer is built to work across all color modes out of the box.
+
+GitHub's UI offers a variety of different color modes. Every pattern in Primer is built to work across all color modes out of the box.
+
+When designing product interfaces in Figma, we recommend to design in light mode. This is best, because the primer components for Figma are only available in light mode.
+To preview your work in other modes, use the [Figma color mode plugin](https://www.figma.com/community/plugin/992128487074360945/Change-Color-Mode).
```

Comment:
```suggestion
When working on product interfaces in Figma, we recommend designing in light mode as our Primer Figma components are only available in light mode.
To preview your work in other modes, use the [Figma color mode plugin](https://www.figma.com/community/plugin/992128487074360945/Change-Color-Mode).
```

Reply:
Hey Katie, while I love this, it moves us from a grade 9 to a grade 14 reading experience. I am trying to keep it lower. Can we break it up into multiple sentences. This always makes it easier to read.

Reply:
>it moves us from a grade 9 to a grade 14 reading experience
Hey @lukasoppermann, I'm a bit confused by the grade reading levels, is there somewhere that outlines what this means? 

@langermank's suggestion on line 23 feels straightforward to me - I don't know if using a conjunction makes the reading level that much more difficult (but I suppose this is subjective)

Reply:
So we are suggesting here: https://primer.style/design/foundations/content#1-keep-it-clear a 7th grade or below level. Which I find very hard to reach. However using https://hemingwayapp.com/ you can get at least to a somewhat simple level.

While we are all somewhat deep in the topic colors, tokens, etc. is pretty complex and new stuff for others. Considering not everybody is an english-native speaker, I do think we should consider making out docs as easy as possible.

Please don't get me wrong, I am happy to change & update the text. But I do feel we should strive for as easy as possible.

Reply:
Ah, I don't think I remembered this reference or didn't know how literally to take it (what defines a 7th grade reading level). That's a fair point.

I don't feel that strongly about this - if you want to keep two sentences I would simplify them both, and small fix to remove the comma in the second sentence: 

```suggestion
When designing product interfaces in Figma, we recommend using light mode. This is best because the Primer Figma components are only available in light mode.
To preview your work in other modes, use the [Figma color mode plugin](https://www.figma.com/community/plugin/992128487074360945/Change-Color-Mode).
```